scripts/build-image.sh

```diff
@@ -24,7 +24,13 @@ function usage {
   echo -e "Environment variable:"
   echo -e " PUSH_TO_REGISTRY     \tPush images to caicloud registry or not, options: Y or N. Default value: ${PUSH_TO_REGISTRY}"
 }
-
+function disable-selinux() {
+  if selinuxenabled && [ "$(getenforce)" = "Enforcing" ]; then
```

Comment:
selinuxenabled - tool to be used within shell scripts to determine if selinux is enabled